### ISSUE TITLE
I'd like to help with this project. 

### ISSUE THREAD

**Issue by jsiegenthaler** [jishi/node-sonos-http-api#833]
Hello @jishi 

I'd like to help.

I can breathe new life into this project by keeping the documentation and dependencies up to date.

I've been programming in various languages for around 30 years, and know what to touch and what not to touch.

Please consider this to be an application to be a contributor with commit capabilities.

Happy to answer any questions you may have.

**Comment by ArthurD** [comment]
I'd volunteer as well.  Am using this "in production" currently, as an interface between Sonos and an Elgato 'Stream Deck' (which is running on a RPi that utilizes 'Virtualhere' USB server software) -- it allows my young daughters to have "princess buttons" that control audio playback in their play areas.  =D

It generally works really well.  But I have to restart the node-sonos-http-api server somewhere between 5-20 times a week when they come running to say "our buttons are broken dad!!!".   

It seems like there are quite a few fixes/PRs pending, likely fixing numerous bugs as well as adding additional functionality.  Unfortunately, between the popularity of this repo & the vast number of forks, it's very difficult to discern what direction to follow -- or if I'm better off forking it myself and selectively merging PRs.

Ideal world, the owner / original maintainer could let 3-5 people in as maintainers, do a one-time configuration of a relatively basic "2+ person review required" workflow for accepting PRs, and let this project re-flourish -- without the owner/original maintainer having to dedicate any of their time (unless so inclined, of course) to the project themselves.

Similar to @jsiegenthaler -- I'd also be more than happy to take on a few hours/week worth of responsibility -- doing PR/code reviews, debugging, general maintenance responsibilities...

It's truly a great tool & a real asset to a LOT of people it seems -- I'd really like to see it get some attention that it definitely needs & deserves, and I think there are many others who feel the same way (and a decent number of whom are qualified & willing/able to commit some of their time to doing so).  

Anyways, throwing my hat in the proverbial ring -- I'm absolutely able & willing to commit some time to maintaining this, similar to @jsiegenthaler -- please feel free to respond here or to reach out directly.  Fingers crossed.

Ty!

**Comment by ccjx** [comment]
@jsiegenthaler @ArthurD It's almost a year since your original comments. The original maintainer was and is active on Github but it seems he has no time to respond to this. Given that other issues (including even npm dependency issues like https://github.com/jishi/node-sonos-http-api/issues/851) are not even addressed, perhaps this might be the right time to fork the project?

**Comment by jsiegenthaler** [comment]
@ccjx : I agree, but I cannot do it alone. Happy to work with others on a fork which can be modernised...
